js/stories.js

```diff
@@ -112,7 +112,6 @@ function showUserStories() {
   } else {
     $ownStories.append(`<h5>Nothing to See Here!</h5>`);
   }
-  $ownStories.show();
 }
```

Comment:
Again, same here: the code change makes sense (we discussed it on the call), but the commit message doesn't make sense. You wrote `add empty in fav.story`. I would write something like `fixed invalid items appearing on favourites list`.